spectral_util/glt/__init__.py

```diff
@@ -0,0 +1,18 @@
+import click
+
+from spectral_util.__main__ import LazyCLI
+
+
+@click.group(
```

Comment:
In general I don't love 'hiding' the command names inside the __init__ files for a new package like this; the farther and more fragmented pieces are, the harder it is for new developers.